app/models/manageiq/providers/automation_manager/configured_system.rb

```diff
@@ -1,2 +1,7 @@
 class ManageIQ::Providers::AutomationManager::ConfiguredSystem < ::ConfiguredSystem
+  virtual_column  :inventory_root_group_name, :type => :string
```

Comment:
Assuming `inventory_root_group` is a relationship, can you add a `:uses => :inventory_root_group` clause here for performance reasons?

Reply:
@Fryguy please re-review